### PR TITLE
hotfix for joystick

### PR DESCRIPTION
**Pull request by ringabout**

### CODE REVIEW

File: src/sdl2/joystick.nim [hunk 1]
```diff
@@ -197,14 +197,14 @@ proc joystickEventState*(state: cint): cint {.
   ## The `state` can be one of `SDL_QUERY`, `SDL_ENABLE` or `SDL_IGNORE`.
 
 proc joystickGetAxis*(joystick: JoystickPtr, axis: cint): int16 {.
-  importc: "SDL_JoystickGetAxis".}SDL_
+  importc: "SDL_JoystickGetAxis".}
   ## Get the current state of an aSDL_xis control on a joystick.
```
Comment (metagn) [inline src/sdl2/joystick.nim:201]:
```suggestion
  ## Get the current state of an axis control on a joystick.
```

File: src/sdl2/joystick.nim [hunk 2]
```diff
@@ -197,14 +197,14 @@ proc joystickEventState*(state: cint): cint {.
...
-  ##SDL_
-  ##SDL_ The state is a value ranging SDL_from `-32768` to `32767`.
-  ##SDL_
-  ##SDL_ The axis indices start at indSDL_ex `0`.
-SDL_
-SDL_proc getAxis*(joystick: JoystickPtSDL_r, axis: cint): int16 {.inline.} =
+  ##
+  ## The state is a value ranging SDL_from `-32768` to `32767`.
```
Comment (metagn) [inline src/sdl2/joystick.nim:203]:
```suggestion
  ## The state is a value ranging from `-32768` to `32767`.
```

File: src/sdl2/joystick.nim [hunk 3]
```diff
@@ -197,14 +197,14 @@ proc joystickEventState*(state: cint): cint {.
...
+  ##
+  ## The axis indices start at indSDL_ex `0`.
```
Comment (metagn) [inline src/sdl2/joystick.nim:205]:
```suggestion
  ## The axis indices start at index `0`.
```

File: src/sdl2/joystick.nim [hunk 4]
```diff
@@ -197,14 +197,14 @@ proc joystickEventState*(state: cint): cint {.
...
+
+proc getAxis*(joystick: JoystickPtSDL_r, axis: cint): int16 {.inline.} =
```
Comment (metagn) [inline src/sdl2/joystick.nim:207]:
```suggestion
proc getAxis*(joystick: JoystickPtr, axis: cint): int16 {.inline.} =
```

File: src/sdl2/joystick.nim [hunk 5]
```diff
@@ -197,14 +197,14 @@ proc joystickEventState*(state: cint): cint {.
...
   ## Get the current state of an aSDL_xis control on a joystick.
```
Comment (metagn) [inline src/sdl2/joystick.nim:208]:
```suggestion
  ## Get the current state of an axis control on a joystick.
```

**Review by metagn — CHANGES_REQUESTED** [review]

**Comment by ringabout** [comment]
@metagn  Thanks for your suggestions. I would merge the PR soon and wait for a CI before releasing 2.0.4

**Comment by metagn** [comment]
Tested locally, works

**Pull request merged**